.envrc

```diff
@@ -0,0 +1,11 @@
+if ! has nix_direnv_version || ! nix_direnv_version 2.2.1; then
+  source_url "https://raw.githubusercontent.com/nix-community/nix-direnv/2.2.1/direnvrc" "sha256-zelF0vLbEl5uaqrfIzbgNzJWGmLzCmYAkInj/LNxvKs="
+fi
+
+nix_direnv_watch_file flake.nix
+nix_direnv_watch_file flake.lock
+if ! use flake . --impure
+then
+  echo "devenv could not be built. The devenv environment was not loaded. Make the necessary changes to devenv.nix and hit enter to try again." >&2
```

Comment:
Am I missing something? There doesn't seem to be a `devenv.nix` in this PR. Is the message off?

Reply:
Yeah that message actually comes from `devenv init`, which creates a default `.envrc` with that incorrect terminology. I removed the message.